frame/referenda/src/migration.rs

```diff
@@ -188,6 +188,14 @@ pub mod test {
 		}
 	}
 
+	fn increment_referendum_index() {
+		ReferendumCount::<T, ()>::mutate(|x| {
+			let r = *x;
+			*x += 1;
+			r
+		});
+	}
```

Comment:
I think you can just do

```suggestion
	fn increment_referendum_index() {
        ReferendumCount::<T, ()>::mutate(|x| x.saturating_inc());
    }
```

so probably don't need `increment_referendum_index` and can do it inline